packages/desktop-gui/src/lib/browser-model.js

```diff
@@ -25,6 +25,14 @@ export default class Browser {
   }
 
   @computed get icon () {
-    return 'chrome'
+    if (this.family === 'chrome') {
+      return 'chrome'
+    }
+
+    if (this.family === 'firefox') {
+      return 'firefox'
+    }
+
+    return 'globe'
```

Comment:
```
return _
.chain(['chrome', 'firefox'])
.find(_.matches(this.family)
.defaultTo('globe')
.value()
```

Reply:
while we're code-golfing, i might as well throw in my implementation...

```js
({
	'chrome': 'chrome'
	'electron': 'chrome'
	'firefox': 'firefox'
})[this.family] || 'globe'
```